gui/control_main.py

```diff
@@ -3832,9 +3897,35 @@ def pixelSelect(self, event):
             self.aux_send_to_server(comm_s)
         if self.threeClickCount == 4:
             self.threeClickCount = 0
+            self.threeClickSignal.emit('0')
             self.click3Button.setStyleSheet("background-color: None")
+            #removing drawing for three cick centering
+            logger.info('Removing 3 click lines')
+            for i in range(len(self.threeClickLines)):
+                self.scene.removeItem(self.threeClickLines[i])
+                logger.info('Removed line {}'.format(i))
+            self.threeClickLines = []
+
         return
 
+    '''
+    Function to check if MD motors are rotating or not
+    '''
+    def omegaMoveCheck(self, sleeptime,call='OmegaState'):
+        state = self.md2.exporter.read(call)
+        while(state == 'Moving'):
```

Comment:
if these kinds of checks are going to be done in multiple areas of the code, I suggest moving the function into a more central place and have the states as enums or constants (like things in config_params.py) so that small issues such as wrong comparison of strings ("string" != "String") don't lead to functional issues.